include/constants/vars.h

```diff
@@ -77,7 +77,7 @@
 #define VAR_REGICE_STEPS_2                   0x403C
 #define VAR_REGICE_STEPS_3                   0x403D
 #define VAR_ALTERING_CAVE_WILD_SET           0x403E
-#define VAR_ALWAYS_ZERO_0x403F               0x403F // This var is read and written, but is always zero.
+#define VAR_DISTRIBUTE_EON_TICKET            0x403F // This var is read and written, but is always zero. Presumably meant to be set externally by Mystery Gift distribution 
```

Comment:
I believe the EON TICKET is only obtainable in emerald via record mixing with a R/S game.

Reply:
Updated comment to remove the speculation/add context. The meaning of the var is still the same, based on its usage in the source